lisa/tools/swap.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+from lisa.tools.lsblk import Lsblk
+from lisa.tools.swapon import SwapOn
+
+
+class Swap(Tool):
+    @property
+    def command(self) -> str:
+        raise NotImplementedError()
+
+    def _check_exists(self) -> bool:
+        return True
+
+    def is_swap_enabled(self) -> bool:
+        swapon = self.node.tools[SwapOn].run("-s").stdout
+        lsblk = self.node.tools[Lsblk].run().stdout
+        if ("swap" in swapon) or ("SWAP" in lsblk):
```

Comment:
Can they be checked one by one? If first tool detects, it doesn't need to run another one.

Reply:
Swaps can be either partition or file and `swapon` should show both these types. But lisav2 code was checking both lsblk and swapon, so I think there might cases which are missed by swapon.

Reply:
check it one by one, it doesn't miss any logic. And it has chance to reduce a remote call.

Reply:
Sorry I misunderstood your comment earlier. I thought this was regarding why are we using both swapon and lsblk together. Will update it to check one by one